package.json

```diff
@@ -181,7 +181,7 @@
     "jest-fail-on-console": "3.1.2",
     "jest-junit": "16.0.0",
     "jest-matcher-utils": "29.7.0",
-    "lerna": "7.4.1",
+    "lerna": "^8.1.2",
```

Comment:
FYI - I believe this update will break package publishing. I also need to upgrade it to land [this PR](https://github.com/grafana/grafana/pull/81873). Maybe best to be tackled in a separate PR first? 🤔